contracts/cw3-fixed-multisig/src/state.rs

```diff
@@ -20,7 +20,7 @@ pub struct Proposal {
     pub title: String,
     pub description: String,
     pub expires: Expiration,
-    pub msgs: Vec<CosmosMsg<Empty>>,
+    pub msgs: Vec<SubMsg<Empty>>,
```

Comment:
Let's leave this as `CosmosMsg` as well. No need to store all the execution parameters.